src/main/scala/org/globalforestwatch/summarystats/firealerts/FireAlertsSummary.scala

```diff
@@ -0,0 +1,112 @@
+package org.globalforestwatch.summarystats.firealerts
+
+import cats.implicits._
+import geotrellis.contrib.polygonal.CellVisitor
+import geotrellis.raster._
+import org.globalforestwatch.summarystats.Summary
+import org.globalforestwatch.util.Util.getAnyMapValue
+import org.globalforestwatch.util.{Geodesy, Mercantile}
+
+/** LossData Summary by year */
+case class FireAlertsSummary(stats: Map[FireAlertsDataGroup, FireAlertsData] =
+                             Map.empty,
+                             kwargs: Map[String, Any])
+  extends Summary[FireAlertsSummary] {
+
+  /** Combine two Maps and combine their LossData when a year is present in both */
+  def merge(other: FireAlertsSummary): FireAlertsSummary = {
+    // the years.combine method uses LossData.lossDataSemigroup instance to perform per value combine on the map
+    FireAlertsSummary(stats.combine(other.stats), kwargs)
+  }
+}
+
+object FireAlertsSummary {
+  // TreeLossSummary form Raster[TreeLossTile] -- cell types may not be the same
+
+  implicit val mdhCellRegisterForFireAlertsRaster
+    : CellVisitor[Raster[FireAlertsTile], FireAlertsSummary] =
+    new CellVisitor[Raster[FireAlertsTile], FireAlertsSummary] {
+
+      def register(raster: Raster[FireAlertsTile],
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method is longer than 50 lines](https://app.codacy.com/manual/thomas-maschler/gfw_forest_loss_geotrellis/pullRequest?prid=5008474)